### PR TITLE
fixed api with_content for :username/bmarks and /bmarks

### PR DESCRIPTION
**Pull request by dyladan**
wrote test to ensure this does not regress

### CODE REVIEW

**Comment by dyladan** [comment]
This fixes #445

File: bookie/views/api.py [hunk 1]
```diff
@@ -409,7 +409,7 @@ def bmark_recent(request, with_content=False):
         return_obj['url'] = res.hashed.url
         return_obj['total_clicks'] = res.hashed.clicks
 
-        if with_content:
+        if 'with_content' in params and params['with_content'] != 'false':
```
Comment (mitechie) [inline bookie/views/api.py]: there's an asbool thing that will help check for false, False, FALSE, etc. Check out http://www.nullege.com/codes/search/pyramid.settings.asbool

I'd prefer we use that to verify that it's true and false. We should have a test for both cases. 

Thanks for the work on this and investigation.

**Comment by mitechie** [comment]
Comment in the diff. Let me know if you need a hang with anything.

**Comment by mitechie** [comment]
Testing out if :sos: will force a test build on another person's branch.

**Comment by dyladan** [comment]
@mitechie switched to use asbool and added a false test

**Comment by mitechie** [comment]
Cool, can you rebase into a single commit and then I'll have it run through CI

**Comment by dyladan** [comment]
rebased and pushed

**Comment by mitechie** [comment]
Hopefully CI is now working with update. :sos:

**Comment by mitechie** [comment]
ok to test

**Comment by mitechie** [comment]
ok to test

**Pull request closed without merging**